docs/README.md

```diff
@@ -27,6 +27,7 @@ Design Docs
 - [.NET Globalization Invariant Mode](design/features/globalization-invariant-mode.md)
 - [WASM Globalization Icu](design/features/globalization-icu-wasm.md)
 - Many more under [design/features](design/features/)
```

Comment:
```suggestion
- Runtime feature designs under [design/features](design/features/)
```
README.md is premium real estate. I do not that links to invariant globalization mode design docs belong here.